baseband/vlbi_base/utils.py

```diff
@@ -122,3 +122,22 @@ def _crc(self, stream):
         for i in range(0, len(stream) - len(self)):
             stream[i:i+pol_bin.size] ^= (pol_bin & stream[i])
         return stream[-len(self):]
+
+
+# PY2
+def gcd(a, b):
```

Comment:
The breadcrumb is OK, but I'd prefer to use the system routine if possible:
```
if not six.PY2:  # ignoring python < 3.5; could add if you wish
    from math import gcd
else:
    def gcd(...):
```

Reply:
In that case let's import `fractions.gcd` if `six.PY2`.